layers/src/main/java/entity/CakeLayer.java

```diff
@@ -22,21 +22,49 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package com.iluwatar.layers.app;
+package entity;
 
-import org.junit.jupiter.api.Test;
 
-import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.CascadeType;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
 
 /**
- *
- * Application test
- *
+ * CakeLayer entity.
  */
-class AppTest {
+@Entity
+@Data
```

Comment:
IntelliJ warns:
`Using @Data for JPA entities is not recommended. It can cause severe performance and memory consumption issues.`

Reply:
done